website/pages/en/publishing/publishing-a-subgraph.mdx

```diff
@@ -2,38 +2,81 @@
 
 <VideoEmbed youtube="satdwdbvzAQ" />
 
-You can find the list of the supported networks [Here](/developing/supported-networks).
+Check out the list of [supported networks](/developing/supported-networks).
+
+## Publishing from Subgraph Studio
+
+1. Go to the [Subgraph Studio](https://thegraph.com/studio/) dashboard
+2. Click on the **Publish** button
+3. Your subgraph will now be visible in [Graph Explorer](https://thegraph.com/explorer/).
+
+All published versions of an existing subgraph can:
+
+- Be published to Arbitrum One. [Learn more about The Graph Network on Arbitrum](/arbitrum/arbitrum-faq).
+
+- Index data on any of the [supported networks](/developing/supported-networks), regardless of the network on which the subgraph was published.
 
-## Publishing a subgraph
+### Updating metadata for a published subgraph
 
-Subgraphs can be published to the decentralized network directly from Subgraph Studio dashboard by clicking on the **Publish** button. Once a subgraph is published, it will be available to view in [Graph Explorer](https://thegraph.com/explorer/).
+- After publishing your subgraph to the decentralized network, you can update the metadata anytime in Subgraph Studio.
+- Once you’ve saved your changes and published the updates, they will appear in Graph Explorer.
+- It's important to note that this process will not create a new version since your deployment has not changed.
 
-- Subgraphs can be published to Arbitrum One. [Learn more about The Graph Network on Arbitrum](/arbitrum/arbitrum-faq).
+## Publishing from the CLI
+
+As of version 0.73.0, you can also publish your subgraph with the `graph-cli`.
+
+1. Open the `graph-cli`.
+2. Use the following commands: `graph publish` then `graph build && graph codegen`
+3. A window will open, allowing you to connect your wallet, add metadata, and deploy your finalized subgraph to a network of your choice.
+
+![cli-ui](/img/cli-ui.png)
 
-- Regardless of the network the subgraph was published on, it can index data on any of the [supported networks](/developing/supported-networks).
+### Customizing your deployment
 
-- When publishing a new version for an existing subgraph the same rules apply as above.
+You can upload your subgraph build to a specific IPFS node and further customize your deployment with the following flags:
+
+```
+USAGE
+  $ graph publish [SUBGRAPH-MANIFEST] [-h] [-i <value>] [--ipfs-hash <value>] [--webapp-url <value>] [--subgraph-id <value>] [--protocol-name <value>]
+
+FLAGS
+  -h, --help            Show CLI help.
+  -i, --ipfs=<value>    [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
+  --ipfs-hash=<value>   IPFS hash of the subgraph manifest to deploy.
+  --webapp-url=<value>  [default: https://cli.thegraph.com/publish] URL of the web UI you want to use to deploy.
+  --subgraph-id=<value>     ID of the subgraph to update.
+  --protocol-name=<value>   Name of the protocol for the subgraph update.
+
+```
```

Comment:
```suggestion
```
USAGE
  $ graph publish [SUBGRAPH-MANIFEST] [-h] [--protocol-network arbitrum-one|arbitrum-sepolia --subgraph-id <value>] [-i <value>] [--ipfs-hash <value>] [--webapp-url
    <value>]

FLAGS
  -h, --help                   Show CLI help.
  -i, --ipfs=<value>           [default: https://api.thegraph.com/ipfs/api/v0] Upload build results to an IPFS node.
  --ipfs-hash=<value>          IPFS hash of the subgraph manifest to deploy.
  --protocol-network=<option>  [default: arbitrum-one] The network to use for the subgraph deployment.
                               <options: arbitrum-one|arbitrum-sepolia>
  --subgraph-id=<value>        Subgraph ID to publish to.
  --webapp-url=<value> 

```
```